src/utils/__tests__/createXArray.test.ts

```diff
@@ -0,0 +1,84 @@
+import { toBeDeepCloseTo } from 'jest-matcher-deep-close-to';
+
+import { createXArray } from '../createXArray';
+
+expect.extend({ toBeDeepCloseTo });
+
+describe('createXArray', () => {
+  it('case when we sample within a specific range with log distribution and include start and end points', () => {
+    let result = createXArray({
+      from: 1,
+      to: 5,
+      length: 5,
+      includeFrom: true,
+      includeTo: true,
+      distribution: 'log',
+      base: 2,
+    });
+    expect(result).toBeDeepCloseTo([2, 4, 8, 16, 32]);
```

Comment:
If from is 1 and to is 5 I'm expecting to have as first element 1 and last element 5.